.github/DISCUSSION_TEMPLATE/q-a.yml

```diff
@@ -0,0 +1,34 @@
+title: "[Question]"
+labels: ["help wanted"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This is the place to ask for specifics about flet or for help with problems you encountered while using it.
```

Comment:
Could you change `flet` to `Flet` please?

Reply:
done